cmd/server/config.go

```diff
@@ -11,9 +10,12 @@ import (
 )
 
 const (
-	ocidRoot            = "/var/lib/ocid"
+	ocidRoot            = "/var/lib/containers"
```

Comment:
I would prefer to leave this containers, since we want the storage to be shared by other container runtimes.